docs/reference/server-events.md

```diff
@@ -956,11 +961,13 @@ Triggered just before the search index page is served, after the page's results
   </template>
 </AposCodeBlock>
 
-## `@apostrophecms/template`
+## `@apostrophecms/template` events
+
+Events emitted by the `@apostrophecms/template` module.
 
 ### `addBodyData`
 
-**Not intended for project use.**  Triggered just before a body data object is added to a request prior to rendering a page.
+**Not intended for project use.**  Triggered just before a body data object is added to a request prior to rendering a page. A module's `getBrowserData` or `enableBrowserData` are better places to work on this data in project code.
```

Comment:
Hmm, this isn't an "or." How about this:

Normally it is better to invoke `self.enableBrowserData()` from your module's `init()` function, optionally passing `'public'` if the data should be available to logged-out site visitors, and implement a `getBrowserData(req)` method which returns the data to be made available as `apos.modules['your-module-name']` in browser code. Most modules will already have one, so use `extendMethods` to make sure you do not lose data required for the editing experience.

... OK, that's a lot of information to put here, so maybe this should be converted to a new doc ticket on how to make data available in the browser at page load time.

Reply:
But we do need to fix that "or".